### PR TITLE
Minor refactor to switch to google-legacy-loader (Fixes #1)

### LINKED ISSUE #1: Remove google-feed-api
It's confusing that's not under 
https://github.com/GoogleWebComponents/google-feeds/blob/0.8-preview/google-feed-api.html

We already have this over at https://github.com/GoogleWebComponents/google-apis/blob/0.8-preview/google-legacy-loader.html

### PR DESCRIPTION
**Pull request by addyosmani**
r: @ebidel wrt #1

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "authors": [
+    "Addy Osmani <addyosmani@gmail.com>"
+  ],
+  "description": "Google Feeds element for Polymer",
+  "main": "google-feeds.html",
+  "keywords": [
+    "web-components",
+    "polymer",
+    "feeds",
+    "feeds",
```
Comment (ebidel) [inline bower.json]: remove

File: bower.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "api"
+  ],
+  "license": "Apache-2",
+  "dependencies": {
+    "polymer": "Polymer/polymer#v0.8.0-rc.7",
```
Comment (ebidel) [inline bower.json:18]: `#v0.9.0-rc.``

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "google-apis": "GoogleWebComponents/google-apis#0.8-preview",
+    "iron-jsonp-library": "PolymerElements/iron-jsonp-library#~0.8.1"
```
Comment (ebidel) [inline bower.json:20]: `#^0.9.0`

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "resolutions": {
+    "paper-styles": "^0.8.3",
+    "iron-flex-layout": "^0.8.0",
```
Comment (ebidel) [inline bower.json:24]: these should be latest as well

**Comment by ebidel** [comment]
Reviewed. just the version bumps

File: bower.json (hunk 2 above)
Comment (addyosmani) [inline bower.json:18]: We're currently stuck on pre-0.9rc until the latest round of gesture bugs get resolved. It's currently busting Starter Kit and paper-drawer-panel :/

**Comment by ebidel** [comment]
Ignore me then until the version can rev.

**Comment by ebidel** [comment]
LGTM

**Comment by addyosmani** [comment]
Thanks for the quick review, Eric!

**Pull request merged**